src/pages/lib/auth/manageusers/q/platform/[platform].mdx

```diff
@@ -1,11 +1,456 @@
+  title: `Manage user profiles`,
+  description: `Learn more about how to enable customers to personalize their profile and verify their contact information with attributes.`,
 };
 
-import manageUsers from "/src/fragments/lib/auth/js/manageusers.mdx";
+<InlineFilter filters={["js", "react-native"]}>
 
-<Fragments fragments={{
-  js: manageUsers,
-  'react-native': manageUsers
-}} />
\ No newline at end of file
+User profile management helps you keep your applications secure while also personalizing your app experiences. In this guide we will review how you can enable your users to personalize their profile and verify their contact information. This includes outlining how you can set up user attributes, verify them, and allow your users to delete them when necessary.
+
+Before you begin, you will need:
+- An Amplify project with the Auth category configured
+- The Amplify libraries installed and configured
+
+## Set up user attributes
+
+Attributes such as name, email address, phone number help you identify individual users. Defining the user attributes you include for your user profiles makes user data easy to manage at scale. This information will help you personalize user journeys, tailor content, provide intuitive account control, and more. You can capture information upfront during sign-up or in a customizable profile. In this section we take a closer look at working with user attributes and how to set them up and manage them.
+
+### Enable standard attributes
+
+Amazon Cognito has a set of default standard attributes. To configure and enable standard user attributes in your app, you can run the Amplify `update auth` command and choose `Walkthrough all the auth configurations`. When prompted for `Specify read attributes` and `Specify write attributes`, choose the attributes you'd like to enable in your app. 
+
+<Accordion title='Review standard attributes options' headingLevel='4' eyebrow='Learn more'>
+
+There are many user attributes available to use by default in Cognito. Their definitions are based on the [OpenID Connect specification](https://openid.net/specs/openid-connect-core-1_0.html#StandardClaims):
+
+- `address`
+- `birthdate`
+- `email`
+- `family_name`
+- `gender`
+- `given_name`
+- `locale`
+- `middle_name`
+- `name`
+- `nickname`
+- `phone_number`
+- `picture`
+- `preferred_username`
+- `profile`
+- `zoneinfo`
+- `updated_at`
+- `website`
+
+You can find a [list of all the standard attributes here](https://docs.aws.amazon.com/cognito/latest/developerguide/user-pool-settings-attributes.html#cognito-user-pools-standard-attributes).
+
+You also have the ability to define custom attributes and up to 50 custom attributes can be added to your user pool. You can review the documentation on [defining custom attributes](https://docs.aws.amazon.com/cognito/latest/developerguide/user-pool-settings-attributes.html#user-pool-settings-custom-attributes) to learn more.
```

Comment:
let us remove the mention of customer user attributes for now please. We need more work to properly support it.